app/models/katello/docker_manifest.rb

```diff
@@ -10,7 +10,8 @@ class DockerManifest < Katello::Model
              :dependent => :delete_all, :inverse_of => :docker_manifest
     has_many :docker_manifest_lists, :through => :docker_manifest_list_manifests, :inverse_of => :docker_manifests
 
-    CONTENT_TYPE = Pulp::DockerManifest::CONTENT_TYPE
+    CONTENT_TYPE = "docker_manifest".freeze
```

Comment:
because the 'content types' need to be pulp-version agnostic, and the Pulp::DockerManifest class is a pulp 2 specific thing.  In most cases the content_type from pulp 2 to pulp 3 will probably be the same.  But something like a 'file' content type is different in pulp 2 vs pulp 3 (iso vs file)